rpc/rpcserver/server.go

```diff
@@ -57,10 +57,10 @@ import (
 
 // Public API version constants
 const (
-	semverString = "5.0.0"
+	semverString = "5.0.1"
```

Comment:
This change is a minor API bump, not patch.  The bump must also be recorded in the api.md document.